tests/Unit/DataStructures/DataBox/Test_DataBox.cpp

```diff
@@ -1049,6 +1049,12 @@ struct TestDataboxMutateApply {
   TestDataboxMutateApply& operator=(TestDataboxMutateApply&&) = default;
   ~TestDataboxMutateApply() = default;
 
+  // These typelists are used by the `db::mutate_apply` overload that does not
+  // require these lists as template arguments
+  using mutate_tags =
```

Comment:
Should consistently use `return_tags`?

Reply:
Perhaps we should discourage the use of the new GitHub "draft" feature because it disables Travis.  (This code doesn't compile.)

Reply:
My view is that "draft" PRs are fine, but it makes absolutely no sense to request reviews on them. I can see a use for them as "Hey, here's code that doesn't compile but I want person X to look at it and help me." I really don't understand why reviews were requested on this PR.

Reply:
I opened this as a draft because of the reason I wrote in the description - I’d like some brief initial feedback that the reviewers agree with the overall design before I polish it, propagate changes through the code, and write additional tests.

Reply:
So before I remove the draft status I’d like to ask (as I wrote in the description) if I should keep the `MutateApply` action in this PR?